_layouts/post.html

```
@@ -0,0 +1,67 @@
+  <div class="analog-clock">
+    <div class="clock-face">
+      <div class="hour-hand"></div>
+      <div class="minute-hand"></div>
+      <div class="second-hand"></div>
+    </div>
+  </div>
+
+  <div class="blog-container">
+    <header class="blog-header">
+      <!-- Mini Clock - Top Left -->
+      <a href="{{ "/" | relative_url }}" class="mini-clock-link">
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)